toonz/sources/tnztools/tooloptions.cpp

```diff
@@ -1887,51 +1887,11 @@ BrushToolOptionsBox::BrushToolOptionsBox(QWidget *parent, TTool *tool,
                              TStroke::OutlineOptions::MITER_JOIN);
   }
   hLayout()->addStretch(1);
-  filterControls();
-}
-
-//-----------------------------------------------------------------------------
-
-void BrushToolOptionsBox::filterControls() {
```

Comment:
I believe the removal of this logic is breaking the Toonz Raster Brush option switching between standard and MyPaint brush options.

I'm not sure if this is really needed if `TPropertyGroup *ToonzRasterBrushTool::getProperties(int idx)` is updated to behave similar to `TPropertyGroup *FullColorBrushTool::getProperties(int targetType)` where it returns the list of controls to display on the tooloptions bar.